assets/javascripts/kitten/components/simulators/loan-simulator.js

```diff
@@ -207,16 +207,33 @@ class LoanSimulatorContent extends React.Component {
       ? this.toCurrency(this.props.commissionAmount * 100)
       : '--'
 
-    const amountToDisplay = ` ${ amount } `
+    let commissionAmount = ' 0 '
+    let exemptionText = null
+
+    if (!this.props.feesExemption) {
+      commissionAmount = ` ${ amount } `
```

Comment:
On pourrait mettre 
```js
commissionAmount = ` ${ amount } ${ this.props.currencySymbol }`
```

Ça nous éviterait de répéter `this.props.currencySymbol ` en dessous.